shimmingtoolbox/optimizer/lsq_optimizer.py

```diff
@@ -47,6 +47,7 @@ def __init__(self, coils: ListCoil, unshimmed, affine, opt_criteria='mse', reg_f
         }
         if opt_criteria in lsq_residual_dict:
             self._criteria_func = lsq_residual_dict[opt_criteria]
+            self.method_to_minimize = opt_criteria
```

Comment:
```suggestion
            self.opt_criteria = opt_criteria
```

You should name the input the same as the attribute name so that it is less confusing